skynet/dgpu/utils.py

```diff
@@ -9,8 +7,10 @@
 import importlib
```

Comment:
above `time` import is unused says the `ruff`

Reply:
Removed in `structify` branch (coming soon)